source/api_documentation/transactions_api/_advertiser_optional_param_table.rst

```diff
@@ -0,0 +1,423 @@
+    - Fees
+    - Advertiser Telecommunications fee associated with transaction
+
+  * - affiliate_campaign_id_from_network
+    - Affiliate Campaign ID
+    - The Campaign ID from the network as set on the affiliate campaign.
+
+  * - bundled_destination_ids
+    - Bundled Destination Ids
+    - 
+
+  * - bundled_destination_ids_from_network
+    - Bundled Destination Ids From Network
+    - 
+
+  * - call_center_call_id
+    - Call Center Call Id
+    - [Call center integration only] The unique identifier assigned to the call by the call center
+
+  * - call_outcome_custom_parameter_1
+    - Call Outcome Custom Param 1
+    - 
+
+  * - call_outcome_custom_parameter_2
+    - Call Outcome Custom Param 2
+    - 
+
+  * - call_outcome_custom_parameter_3
+    - Call Outcome Custom Param 3
+    - 
+
+  * - call_outcome_description
+    - Call Outcome Description
+    - [Outcome API only] The description of a Call Outcome, can contain alphanumeric characters.
+
+  * - call_outcome_occurred_at
+    - Call Outcome Occurred At
+    - The timestamp that the Call Outcome occurred (defaults to when the Call Outcome was reported to Invoca), but when reporting via API the customer can override with a specific time.
+
+  * - call_outcome_outcome_name
+    - Outcome Name
+    - The name of the Outcome that was reported.
+
+  * - call_outcome_partner_unique_id
+    - Call Outcome Partner ID
+    - [Outcome API only] Unique id from customer's system associated to the Call Outcome
+
+  * - call_outcome_source
+    - Call Outcome Source
+    - The source of the Call Outcome ('Api' or 'Ivr').
+
+  * - call_outcome_value
+    - Call Outcome Value
+    - Whether the outcome was met, 1 = outcome met, 0 = outcome not met (defaults to 1).
+
+  * - cell_phone_number
+    - Cell Phone (Reported)
+    - [Call center integration only] Customer's cell phone number as given to the call center.
+
+  * - country
+    - Country (Reported)
+    - [Call center integration only] Customer's country as given to the call center.
+
+  * - dynamic_number_pool_id
+    - Pool ID
+    - [Number pooling links only] The ID of the pool.
+
+  * - dynamic_number_pool_pool_type
+    - Pool Type
+    - [Number pooling links only] The type of pool: "Search", "SearchKeyword" or "Custom"
+
+  * - dynamic_number_pool_referrer_ad
+    - AdWords Ad
+    - AdWords Ad Headline copy, provided by Google
+
+  * - dynamic_number_pool_referrer_ad_group
+    - AdWords Ad Group
+    - AdWords Ad Group name, provided by Google
+
+  * - dynamic_number_pool_referrer_ad_group_id
+    - AdWords Ad Group ID
+    - AdWords Ad Group ID, provided by Google
+
+  * - dynamic_number_pool_referrer_ad_id
+    - AdWords Ad ID
+    - AdWords Ad ID, provided by Google
+
+  * - dynamic_number_pool_referrer_param1
+    - Dynamic Number Pool Referrer Param1 Value
+    - [Number pooling links only] name=value for parameter 1, url escaped. Example: name%3Dvalue
+
+  * - dynamic_number_pool_referrer_param10
```

Comment:
similar theme here.. I don't think strict alphabetical makes sense here.. I would show param1, param2, ... param10 then the 11 ~ 25 in numeric order after that (not sure if we need a header for the 11~25 params or not?)